opm/simulators/flow/Main.hpp

```diff
@@ -189,84 +158,12 @@ class Main
     }
 
 private:
```

Comment:
we can maybe move this block below so we don't need two private: ?

Reply:
> we can maybe move this block below so we don't need two private: ?

Good idea.  I've pushed an update to that effect and, while here, also added some skeletal documentation to `(run|dispatch)(Dynamic|Static)`.